core/domain/app_feedback_report_constants_test.py

```diff
@@ -0,0 +1,28 @@
+# coding: utf-8
+#
+# Copyright 2022 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Unit tests for core/domain/app_feedback_report_constants.py"""
+
+from __future__ import annotations
+
+from core.tests import test_utils
+from . import app_feedback_report_constants # pylint: disable=unused-import
+
+
+class AppFeedbackReportConstantsTests(test_utils.GenericTestBase):
+
+    def test_trivial(self) -> None:
```

Comment:
In these files, either add a comment if it makes sense to keep the files like this in the future (that explains the broader picture of why we are doing this), or a TODO if these tests will need to be replaced.

Btw, thanks for creating actual test files for some of the easier ones -- this helps reduce the churn!

Reply:
Yes sure. I'll add proper comments.

Reply:
Hi Sean! Wanted your review on the comment to be added. Does `Our backend per-file coverage checks do not include the files which lack an associated test file. It is essential to ensure that every backend file has an associated test file so that no file can sneak past our coverage checks. However, files like this do not have any logic to be tested, so we create trivial tests for them. ` sounds good?

Reply:
It sounds good. I have added the comments now. PTAL.